src/test/TEST_bench/benchmark_ij.perf.saved.tioga

```diff
@@ -62,16 +62,16 @@ PCG Solve wall clock time = 0.177456 seconds
 PCG Setup wall clock time = 0.240707 seconds
 PCG Solve wall clock time = 0.189251 seconds
 # Output file: benchmark_ij.out.22
-PCG Setup wall clock time = 0.239942 seconds
+PCG Setup wall clock time = 0.291317 seconds
```

Comment:
Timings are about 5% slower now because these tests are using the rocSPARSE's SpGEMM path, which runs CSR sorting